src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue

```diff
@@ -103,22 +103,33 @@ export default {
 	},
 
 	emits: [
+		'has-error',
 		'has-resources',
 	],
 
 	data() {
 		return {
 			appEnabled: OC?.appswebroots?.related_resources !== undefined,
 			headerTranslated: t('Related resources'),
-			descriptionTranslated: t('Anything shared with the same group of people will show up here'),
 			loading: false,
+			error: null,
 			resources: [],
 		}
 	},
 
 	computed: {
 		isVisible() {
-			return !this.loading && this.resources.length > 0
+			if (this.loading) {
+				return false
+			}
+			return this.error ?? this.resources.length > 0
+		},
+
+		description() {
+			if (this.error) {
+				return t('Error getting related resources. Please contact your system administrator if you have any questions.')
```

Comment:
From my side of things this is not correct. When the 400 error occurs with Talks 1-1 chats there is nothing I as sysadmin was able to do about it.

Reply:
Further details in the comments over there: https://github.com/nextcloud/spreed/pull/9256#issuecomment-1497089458

Now that logging is added it actually makes sense, but i find it weird that the error is not using the normal showError anymore